Assets/Code/PersonFollowBoat.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PersonFollowBoat : MonoBehaviour
+{
+    public Rigidbody boat;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+       transform.position = new Vector3(boat.position.x, boat.position.y + 0.25f, boat.position.z);
```

Comment:
see if you can clone and modify vector3 or something

Reply:
also i see what youre saying about this instead of making a new Vector3 but this is the most efficient way to do this. the C# garbage collector will clean up the new objects created, especially the ones created in the Start() function. the C# garbage collector is amazing

Reply:
i mean more like

```cs
var foo = new Vector3(boat.position);
foo.y += 0.25f;
```

not too concerned about performance but rather copying x, y and z manually

Reply:
Unfortunately i cant do that because the x y and z coordinates are non-assignable for some dumb reason

Reply:
hmm have you tried something like

```cs
var foo = boat.position + new Vector3(0, 0.25f, 0);
```

if this doesn't work, then whatever

Reply:
I mean that will work but it takes up more space and processing to do it that way. It is cleaner though. I don't think it matters too much because the person is just a placeholder. Once we have a boat and person model we can make it more realistic and clean

Reply:
i'm not really sure that we should be thinking of performance rn, but that's fine. just make a helper function in the future if you do more vector additions like this, so that we don't have to repeat ourselves.